test.sh

```diff
@@ -30,6 +30,18 @@ echo "running test suit consisting of ${test_list}"
 
 pass=0
 
+testcount=0
+failcount=0
+
+# Prep the results.xml file
+echo '<?xml version="1.0" encoding="UTF-8"?>
+<testsuites>
+   <testsuite name="CI Results" tests="NUMTESTS" failures="NUMFAILURES">' > results.xml
+
+# Prep the results.markdown file
```

Comment:
we should add the job name here as well, so that we know which job the fail/pass is associated with.

Reply:
This corresponds to the overall ci test so can only show total failures. The details get saved in the ci system which shows which specific ones failed. Additionally, a markdown of the actual pass/fails gets dropped into the PR as a comment. So hopefully that should cover it.